.github/workflows/antsibull-docs.yml

```diff
@@ -30,34 +30,31 @@ jobs:
         with:
           path: antsibull-core
 
-      # nb: this is the first version of antsibull-docs that declares support
-      # for antsibull-core v2
```

Comment:
I think this comment should stay so we don't have to re-invent it later. (Same in the next file.)